src/SingleCellAnnotator.imjoy.html

```diff
@@ -321,17 +328,32 @@
     },
     async uploadAnnotation(){
       // Default options are marked with *
+      const sendData = JSON.parse(JSON.stringify(sendDataTemplate));
+      sendData.discard = this.current_image_data.discarded;
+      sendData.mask = this.current_image_data.mask_geojson.features;
+      sendData.bbox = this.current_image_data.mask_geojson.bbox;
+      sendData.location = {};
+      if (this.current_image_data.label_display_mode == 'single'){
+        sendData.location[this.current_image_data.location] = this.current_location_selected;
```

Comment:
You can just send all the locations in current image data

Reply:
should not this be a dict format? for weak annotation, currently, if return this.current_image_data.location, it will always be an organelle name. so I convert that to a dict here

Reply:
> You can just send all the locations in current image data

should not this be a dict format? for weak annotation, currently, if return this.current_image_data.location, it will always be an organelle name. so I convert that to a dict here

Reply:
I see, you are right.